Lib/glyphsLib/types.py

```diff
@@ -281,15 +281,29 @@ def strftime(self, val):
 
 
 def parse_color(src=None):
+    # type: (Optional[str]) -> Optional[Union[Tuple[int, ...], int]]
+    """Parse a string representing a color value.
+    
+    Color is either a fixed color (when coloring something from the UI, see
+    the GLYPHS_COLORS constant) or a list of the format [u8, u8, u8, 1],
+    negative numbers are abs()'d. Glyphs does not support an alpha channel
```

Comment:
> negative numbers are abs()'d

negative numbers are never written to file! this function is supposed to parse a color spec from a string found in a .glyphs file, no? So why should it handle this degenerate case?